lib/wrap-queries.js

```diff
@@ -19,8 +29,10 @@ function wrap() {
 
       // e.g. print.css
       if (name === 'print') {
+        hasAtRule(src);
         contents = new Buffer('\n@media print {\n' + src + '}');
       } else if (!isNaN(startingNumber)) {
+        hasAtRule(src);
```

Comment:
We can't really let their @-rule through, since having nested media queries will surely break (any post-processors like autoprefixer or cssmin will choke on it, and it won't render correctly in the browser).